src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-trash-list/rbd-trash-list.component.ts

```diff
@@ -145,6 +146,11 @@ export class RbdTrashListComponent implements OnInit {
       }
       viewCacheStatusMap[pool.status].push(pool.pool_name);
       images = images.concat(pool.value);
+      if(images.length > 0){
```

Comment:
```suggestion
      if (images.length > 0){
```
I think tslint will complain this, too.

Reply:
Done, though tslink was not complaining

Reply:
Done, tslint was showing this